addons/copy-reporter/addon.json

```
@@ -0,0 +1,20 @@
+{
+  "name": "Copy reporter values",
+  "description": "Adds buttons to copy the value of reporter balloons and monitors.",
+  "credits": [{ "name": "TheKodeToad" }],
+  "tags": ["editor", "codeEditor"],
+  "userscripts": [
+    {
+      "url": "userscript.js",
+      "matches": ["projects"]
+    }
+  ],
+  "userstyles": [
+    {
+      "url": "style.css",
+      "matches": ["projects"]
+    }
+  ],
+  "dynamicEnable": true,
+  "dynamicDisable": true
```

Oh, and to top it off, we need a `versionAdded` attribute here. This will be...
```suggestion
  "dynamicDisable": true,
  "versionAdded": "1.31.0"
```